supertokens_python/recipe/thirdparty/asyncio/__init__.py

```diff
@@ -30,39 +32,50 @@ async def get_user_by_id(
 
 
 async def get_users_by_email(
-    email: str, user_context: Union[None, Dict[str, Any]] = None
+    email: str,
+    tenant_id: Optional[str],
```

Comment:
set it to none by default.. check all places.

Reply:
Okay